ui/common/src/main/res/values/strings.xml

```diff
@@ -320,6 +323,8 @@
   <string name="transaction_type_funds_sent">Funds Sent</string>
   <string name="transaction_type_funds_received">Funds Received</string>
   <string name="transaction_type_gift_card">Gift Card</string>
+  <string name="transaction_rejected_body">This purchase has been rejected.\nFor more information contact support.</string>
```

Comment:
might be better not to change the strings files manually, as this will likely cause problems on Crowdin